### PR TITLE
[pyqt/es-es] PyQT Spanish translation

### PR DESCRIPTION
**Pull request by adrianrocamora**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: es-es/pyqt-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+category: tool
+tool: PyQT
+filename: learnpyqt.py
+contributors:
+    - ["Nathan Hughes", "https://github.com/sirsharpest"]
+translators:
+    - ["Adrian Rocamora", "https://github.com/adrianrocamora"]
+---
+
+**Qt** es un sistema altamente reconocido que permite desarrollar software multiplataforma que puede correr en diferentes entornos de software y hardware con pocos o ningún cambio. Aun así conserva la velocidad y poder de una palicacion nativa. **Qt** fue originalmente escrito en *C++*.
```
Comment (Menelion) [inline es-es/pyqt-es.html.markdown]: aplicación nativa

File: es-es/pyqt-es.html.markdown [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+```python
+import sys
+from PyQt4 import QtGui
+	
+def window():
+    # Crear el objeto de la aplicación
+    app = QtGui.QApplication(sys.argv)
+    # Crear un widget en el que colocaremos nuestra etiqueta
+    w = QtGui.QWidget()
+    # Agregamos nuesta etiqueta al widget
```
Comment (Menelion) [inline es-es/pyqt-es.html.markdown]: nuestra

File: es-es/pyqt-es.html.markdown [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    w.show()
+    # Ejecutemos lo que hayamos solicitado ya inicializado el resto
+    sys.exit(app.exec_())
+
+if __name__ == '__main__':
+    window()
+
+```
+
+Para poder hacer uso de las funciones más avanzades en **pyqt** necestiamos agregar elementos adicionales.
```
Comment (Menelion) [inline es-es/pyqt-es.html.markdown]: necesitamos

File: es-es/pyqt-es.html.markdown [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+Aquí mostramos cómo introducir una caja de dialogo popup, útil para permitir al usuario confirmar su decisión o para brindarnos información.
```
Comment (Menelion) [inline es-es/pyqt-es.html.markdown]: diálogo

**Comment by adrianrocamora** [comment]
Awesome! Thanks for bringing these issues to my attention, I've resolved them. Please let me know if there's anything else. Have a great week

**Comment by divayprakash** [comment]
@adrianrocamora Any reason why you closed this without letting it be merged?

**Comment by adrianrocamora** [comment]
Ah, sorry I didn't mean to. I'm reopening it. Thanks!

**Comment by divayprakash** [comment]
@adrianrocamora No worries.

@Menelion This is ready for review!

**Comment by Menelion** [comment]
¡Muchas gracias!

**Comment by adrianrocamora** [comment]
¡Fue un placer!

**Pull request merged**